com.unity.render-pipelines.universal/Editor/UniversalAnalytics.cs

```diff
@@ -37,7 +37,7 @@ static void SendUniversalEvent()
             if (!EnableAnalytics())
                 return;
 
-            // Needd to check if this isn't null
+            // Need to check if this isn't null
             UniversalRenderPipelineAsset rendererAsset = GraphicsSettings.currentRenderPipeline as UniversalRenderPipelineAsset;
```

Comment:
Ah we have something similar to `HDRenderPipeline.currentAsset` 

UniversalRenderPipeline.asset (I guess this was added before the quality settings one so hence the name). maybe we could align and rename this to `currentAsset`. It would make more sense.
https://sourcegraph.com/github.com/Unity-Technologies/Graphics@ba52521a7a0d7015b9d34e0f9d8fa47664ff23f1/-/blob/com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs#L465-468

Reply:
Do you want me to redo this line and change `GraphicsSettings.currentRenderPipeline as UniversalRenderPipelineAsset;` to `UniversalRenderPipeline.asset` ? It could be a good idea

Reply:
I'd appreciate if you could do it, but will not block your PR as this is not introduced in your PR.